spiceaidocs/docs/components/data-connectors/clickhouse.md

```diff
@@ -4,36 +4,66 @@ sidebar_label: 'Clickhouse Data Connector'
 description: 'Clickhouse Data Connector Documentation'
 ---
 
-## Federated SQL query
-
-To connect to any Clickhouse database as connector for federated SQL query, specify `clickhouse` as the selector in the `from` value for the dataset.
+ClickHouse is a fast, open-source columnar database management system designed for online analytical processing (OLAP) and real-time analytics. This connector enables federated/accelerated SQL queries on top of a Clickhouse server.
 
 ```yaml
 datasets:
-  - from: clickhouse:path.to.my_dataset
+  - from: clickhouse:my.dataset
     name: my_dataset
 ```
 
 ## Configuration
 
+### `from`
+
+The `from` field for the Clickhouse connector takes the form of `from:db.dataset` where `db.dataset` is the path to the Dataset within Clickhouse. In the example above it would be `my.dataset`.
```

Comment:
```suggestion
The `from` field for the ClickHouse connector takes the form of `from:db.dataset` where `db.dataset` is the path to the Dataset within ClickHouse. In the example above it would be `my.dataset`.
```